js/script.js

```diff
@@ -0,0 +1,134 @@
+/*  Anna Sullivan
+    script.js
+    INFO2134WW
+    Thoendel
+    5/2/2020
+*/
+window.addEventListener('load', () => {//event listener on page load
+    
+    startClock();
+        //YOUR CODE SHOULD START BELOW THIS LINE
+    
+    navigator.geolocation.getCurrentPosition(findLocation);//call getCurrentPosition using navigator and geolocation
+        
+    function findLocation(position){//function to find position of user
+       console.log(`Latitude: `+ position.coords.latitude `, Longitude: `+ 
+       position.coords.longitude +`.`);

```

Comment:
You are mixing string and template syntax here - so this isn't going to work. And you are missing a +. Finally - really - we need to store lat and long for use later - not just output them  here.

Are you working with another student? These errors are very similar to another students. If you are working together - that should be disclosed in the comments of both submissions. Going forward - I'll have to dock if that isn't declared.